GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionGroupRestService.java

```diff
@@ -173,11 +156,7 @@
         QuestionGroup s = questionGroupDao.getByKey(id);
         QuestionGroupDto dto = null;
         if (s != null) {
-            dto = new QuestionGroupDto();
-            DtoMarshaller.copyToDto(s, dto);
-            // needed because of different names for description in
-            // questionGroup and questionGroupDto
-            dto.setDescription(s.getDesc());
+            dto = QuestionGroupDtoMapper.transform(s);
```

Comment:
I was a little thrown off when I saw the variable name `s`. I thought you were using the function to transform a survey.  May be an idea to rename it appropriately.